splunk_add_on_ucc_framework/ucc_ui_lib/package.json

```diff
@@ -10,7 +10,6 @@
     "eslint": "eslint src --ext \".js,.jsx\"",
     "eslint:ci": "yarn run eslint -f junit -o test-reports/lint-results.xml",
     "eslint:fix": "eslint src --ext \".js, .jsx\" --fix",
-    "link:app": "ln -s $PWD/stage $SPLUNK_HOME/etc/apps/my-splunk-app",
```

Comment:
@harshpatel-crest Can we keep this script. I'm using this for the testing.

Reply:
Sure.